app/app.rb

```diff
@@ -6,10 +8,13 @@
 require_relative '../person/teacher'
 
 class App
+  DATA_DIR = 'data/'.freeze
+
   def initialize
     @books = []
     @people = []
     @rentals = []
+    load_data
```

Comment:
- [ ] Kindly take note that I was unable to list the data (options 1 and 2) stored in both `books.json` and `people.json` files. I got the error message screenshot below. 👇 

**Note: Please kindly ensure that `list rentals option` also display the data from rentals.json file as well. I couldn't check the functionality because of the issue highlighted above.** Thanks.

<img width="927" alt="Untitled" src="https://github.com/ahmidou09/OOP-school-library/assets/103386183/5464f222-697d-416c-b6fa-2e10457d339f">